app/main.py

```diff
@@ -1 +1,9 @@
-# write your code here
+def copy_file(file_name: str) -> None:
+    words_of_file_name = file_name.split(" ")
```

Comment:
`" "` is default value for split
```suggestion
    words_of_file_name = file_name.split()
```